index.js

```diff
@@ -1 +1,7 @@
-// Write your code here!
\ No newline at end of file
+const mainNode = document.getElementById('main')
+mainNode.remove()
+
+const newHeader = document.createElement('h1');
+newHeader.setAttribute('id', 'victory')
+const myName = 'Kyle'
+newHeader.textContent = `${myName} is the champion`
```

Comment:
Any reason for making your name a variable? 
```suggestion
newHeader.textContent = `Kyle is the champion`
```

Reply:
funsies